v63002/u_inref/gtm8760.csh

```diff
@@ -0,0 +1,34 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Testing a long string as a subscript and a value for global and local variables
+# All of these would produce core files in previous versions when the string is at least 32768 characters
+#
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+foreach n (32766 32767 32768 32769)
```

Comment:
Add 1048576 and 1048577 (i.e. 1Mb and 1Mb+1) to test that upto 1Mb works fine except as a subscript in globals. You will need to set the record_size to 1Mb (-record_size parameter to dbcreate.csh).

Reply:
For the record, we tried the 1Mb suggestion but we ran into issues with the shell "Arg list too long" errors when running anything afterwards. So abandoned it.